.github/ISSUE_TEMPLATE/office-hours-template.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Just curious, how would this file be used?

Reply:
How this template can be picked up?
Github seems only pick ISSUE_TEMPLATE.md file.
Can we enable Wiki on Github and allow people sign-up there? Wiki is also a git, we should be able to look at the wiki update history if there is abuse of it.